src/fdc3/dotnet/DesktopAgent/src/DesktopAgent/UserChannel.cs

```diff
@@ -53,27 +50,34 @@ public async ValueTask Connect()
             throw new ObjectDisposedException(nameof(UserChannel));
         }
 
-        await _desktopAgentService.ConnectAsync(CancellationToken.None);
+        await _messageRouter.ConnectAsync();
+
+        var broadcastObserver = AsyncObserver.Create<TopicMessage>(x => HandleBroadcast(x.Payload));
 
-        var broadcastSubscribing = _desktopAgentService.SubscribeAsync(_topics.Broadcast, HandleBroadcast, CancellationToken.None);
+        var broadcastSubscribing = _messageRouter.SubscribeAsync(_topics.Broadcast, broadcastObserver);
 
-        await _desktopAgentService.RegisterServiceAsync<GetCurrentContextRequest>(_topics.GetCurrentContext, GetCurrentContext);
-        
+        //await _messageRouter.RegisterEndpointAsync(_topics.GetCurrentContext, GetCurrentContext);
```

Comment:
Any reason for leaving this commented instead of releting?

Reply:
This is the original file, I just reverted back that:
https://github.com/morganstanley/ComposeUI/blob/main/src/fdc3/dotnet/DesktopAgent/src/DesktopAgent/UserChannel.cs#L59

Reply:
Feel free to fix my mistakes ;)